pkgs/development/python-modules/pdfrw/default.nix

```
@@ -0,0 +1,20 @@
+{ stdenv, buildPythonPackage, fetchPypi, mupdf, swig }:
+buildPythonPackage rec {
```

add a line here as well :)
```suggestion

buildPythonPackage rec {
```